bin/pint

```diff
@@ -113,7 +125,7 @@ for csp in cloud_providers:
 
 image_state = None
 image_states_data = ifsrequest.get_image_states_data(
-    None, None, 'json', None
+    None, None, 'json', 'all', 'bz2', None
```

Comment:
Note that you are adding the adding the `'all'` parameter in here that was previously being specified as `None` which would appear to have been a bug.

As suggested above it may be better (and more Pythonic) to removed arguments that specify default values, and used named paramater assignment for non-default value specifications:

```suggestion
    None, None, result_format='json'
```